Packages/vcs/Lib/vcsvtk/boxfillpipeline.py

```diff
@@ -0,0 +1,383 @@
+from .pipeline2d import Pipeline2D
+from .. import vcs2vtk
+
+import numpy
+import vcs
+import vtk
+import warnings
+
+class BoxfillPipeline(Pipeline2D):
+    """Implementation of the Pipeline interface for VCS boxfill plots.
+
+    Internal variables:
+        - self._legend: Contour labels.
```

Comment:
oh I see we're in boxfill... yes this is probably the strings/labels to draw on the legend, user-specified. I guess it's an ok name in this case, since we're talking about labels on legend....